### ISSUE TITLE
Shrink apt.exe as much as possible

### ISSUE THREAD

**Issue by maphew** [maphew/apt#53]
Using pyinstaller with out of the box settings works, but the resulting file is really big: 9mb when bundled as _onefile_, and 20mb when as _onedir_ (!)

Inspecting the expanded folder I see Tk and Tcl are included. We certainly don't need those modules, and I suspect there are probably others that we can exclude too.

**Comment by maphew** [comment]
Oh look, I've been here before, 9 years ago. Eep. From <https://groups.google.com/forum/#!topic/pyinstaller/gDDIOhtdrjA>:

> Have a look at where you're wasting space by running ArchiveViewer.py 
> over the resulting executable.
> 
> Try with older versions of Python, like 2.3. python.dll grew a lot in 
> 2.4 and 2.5, and maybe you don't need all those new features. Another 
> solution is to compile yourself a stripped down version of python.dll. 
> It's pretty easy (there's a single file where all the builtin modules 
> are linked: just comment out the ones you don't need). I have been using 
> a stripped down version of python25.dll for a long time, which was much 
> less than a half than the original version and still be very useful for 
> most projects.
> -- 
> Giovanni Bajo
> Develer S.r.l.
> http://www.develer.com

ArchiveViewer is part of pyinstaller. I don't think we want to go to older python versions, but the custome compiled .dll idea might be interesting. However before going that far lets see what we can gain using Pyinstaller's built in excludes, and this recipe:
https://github.com/pyinstaller/pyinstaller/wiki/Recipe-remove-tkinter-tcl

**Comment by maphew** [comment]
With _onedir_ apt.exe I moved `m*.dll` out of the way. On my machine all apt.exe commands worked. 

I then repeated the same test on a virtual machine (Win7 Enterprise SP1). Here console reported `Error loading Python DLL: V:\work\apt-exe\python27.dll (error code 126)` and also a popup:

> *apt.exe - System Error* The program can't start because MSVCR90.dll is missing from 
> your computer. Try reinstalling the program to fix this problem. 

I restored `msvcr90.dll` and then proceeded to conduct a virgin osgeo4w install and ran through all apt commands -- no more errors! This .dll is only 640kb. So it looks like we might be able to reduce the distributed apt-exe to 5mb zipped / 9mb expanded.
